generators/app/templates/.terraform-version

```diff
@@ -1,2 +1,2 @@
-<% if (tfVersion == '11') { %>0.11.14<% } -%>
-<% if (tfVersion == '12') { %>0.12.24<% } -%>
+<% if (tfVersion == '11') { %>latest:^0.11<% } -%>
```

Comment:
Same as previous question. Why remove 12 if we offer 11 and 13. Shouldn't we offer 11, 12, and 13? Or should we just offer 12 and 13? Or just 13?